Make for TARGET_WEB is broken

**Describe the bug**

Edit: ~This appears to dupe #96~ The suggested fix from there doesn't work.

Attempting to build with TARGET_WEB=1 results in an error:
```
src/game/mario_actions_cutscene.c:274:17: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
                exit(0);  // exit after saving game
                ^
src/game/mario_actions_cutscene.c:274:17: note: include the header <stdlib.h> or explicitly provide a declaration for 'exit'
1 error generated.
Makefile:796: recipe for target 'build/us_web/src/game/mario_actions_cutscene.o' failed
make: *** [build/us_web/src/game/mario_actions_cutscene.o] Error 1
make: *** Waiting for unfinished jobs....
1 warning generated.
```

**To Reproduce**
```shell
make VERSION=us TARGET_WEB=1 -j4
```

**Expected behavior**
A clean build.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04

**Additional context**
The build fails in exactly the same way using the Docker container.

I ran a `git bisect` on the code to try to figure out when this problem was introduced, and came up with [this commit](https://github.com/sm64pc/sm64pc/commit/691d6f382bb3b5c3feb8f4026d3b921e2ce492c7):

```
✘-2 /opt/sm64pc [:691d6f3|BISECTING|…1] 
00:20 $ git bisect bad
691d6f382bb3b5c3feb8f4026d3b921e2ce492c7 is the first bad commit
commit 691d6f382bb3b5c3feb8f4026d3b921e2ce492c7
Author: Hyenadae <Hyenadae@github.com>
Date:   Mon May 11 06:58:33 2020 -0400

    WIP In-Game Exit via pause+save menus w/ Localizations

:040000 040000 4822f344dd9e4b0064432b03df46832a70564c9d 56853ef10793d5bf3b02dd699b1431989ec40265 M	include
:040000 040000 ae163095a105d81a44b6fab1b77040d6a9512846 f23a7c1bc8a53903b81bcfd9fec3defdf75d64d1 M	src
```

@fgsfdsfgs Causes a different error now:

```
✘-2 /opt/sm64pc [master|…1] 
15:08 $ git rev-parse HEAD
094f3ebb33bb202908b2054e1a5566b077a39b35
✔ /opt/sm64pc [master|…1] 
15:08 $ source emsdk/emsdk_env.sh; make clean; make TARGET_WEB=1 -j4
Setting environment variables:
EMSDK = /opt/sm64pc/emsdk
EM_CACHE = /opt/sm64pc/emsdk/upstream/emscripten/cache

rm -f -r build
make: Entering directory '/opt/sm64pc/tools'
make: Nothing to be done for 'all'.
make: Leaving directory '/opt/sm64pc/tools'
Makefile.split:152: build/us_web/level_rules.mk: No such file or directory
cpp -P -DVERSION_US -DNON_MATCHING -DAVOID_UB -DTARGET_WEB -I . -o build/us_web/level_rules.mk levels/level_rules.mk
<snip>
emcc -c -O2 -g4 --source-map-base http://localhost:8080/ -I include -I build/us_web -I build/us_web/include -I src -I . -DVERSION_US -DNON_MATCHING -DAVOID_UB -DTARGET_WEB -DF3DEX_GBI_2E -fno-strict-aliasing -fwrapv -s USE_SDL=2 -DEXT_OPTIONS_MENU -o build/us_web/data/behavior_data.o data/behavior_data.c
src/pc/cliopts.c:14:5: error: implicitly declaring library function 'strncpy' with type 'char *(char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
    strncpy(gCLIOpts.ConfigFile, "sm64config.txt", strlen("sm64config.txt"));
    ^
src/pc/cliopts.c:14:5: note: include the header <string.h> or explicitly provide a declaration for 'strncpy'
src/pc/cliopts.c:14:52: error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
    strncpy(gCLIOpts.ConfigFile, "sm64config.txt", strlen("sm64config.txt"));
                                                   ^
src/pc/cliopts.c:14:52: note: include the header <string.h> or explicitly provide a declaration for 'strlen'
src/pc/cliopts.c:22:8: error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
                        if (strcmp(argv[i], "--skip-intro") == 0) // Skip Peach Intro
                            ^
src/pc/cliopts.c:22:8: note: include the header <string.h> or explicitly provide a declaration for 'strcmp'
src/pc/cliopts.c:41:17: error: implicitly declaring library function 'strncmp' with type 'int (const char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
            if (strncmp(argv[i], "--configfile", strlen("--configfile")) == 0)
                ^
src/pc/cliopts.c:41:17: note: include the header <string.h> or explicitly provide a declaration for 'strncmp'
src/pc/cliopts.c:48:7: error: implicitly declaring library function 'memset' with type 'void *(void *, int, unsigned long)' [-Werror,-Wimplicit-function-declaration]
                                                memset(gCLIOpts.ConfigFile, 0, 30);
                                                ^
src/pc/cliopts.c:48:7: note: include the header <string.h> or explicitly provide a declaration for 'memset'
5 errors generated.
Makefile:796: recipe for target 'build/us_web/src/pc/cliopts.o' failed
make: *** [build/us_web/src/pc/cliopts.o] Error 1
make: *** Waiting for unfinished jobs....
```

--- a/src/pc/cliopts.c
+++ b/src/pc/cliopts.c
@@ -2,6 +2,7 @@
 #include <strings.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <string.h>
 
 struct PCCLIOptions gCLIOpts;
 

